coralproject/talk
# [CORL-2742] Make dashboard staff counts include admins, mods, site mods, and staff

## What does this PR do?

Updates dashboard count metrics to include admins, mods, site mods, and staff comment counts in its calculations.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

No changes.

## Does this PR introduce any new environment variables or feature flags? 

No new feature flags.

## If any indexes were added, were they added to `INDEXES.md`?

No new indexes.

## How do I test this PR?

- Go to `Admin > Dashboard`
    * Check the amount of `Staff + comments` you have
- Make some staff/admin/mod comments using various roles on a stream
- Go to `Admin > Dashboard`
- Check that comment counts for `Staff + comments` matches the counts you created
 
## How do we deploy this PR?

No special considerations.
